lib/qonto/model/transaction.rb

```diff
@@ -1,20 +1,10 @@
 module Qonto
   module Model
     class Transaction < Base
-      attr_accessor :amount
-      attr_accessor :amount_cents
-      attr_accessor :local_amount
-      attr_accessor :local_amount_cents
-      attr_accessor :side
-      attr_accessor :operation_type
-      attr_accessor :currency
-      attr_accessor :local_currency
-      attr_accessor :settled_at
-      attr_accessor :label
-      attr_accessor :status
-      attr_accessor :transaction_id
-      attr_accessor :emitted_at
-      attr_accessor :note
+      attr_accessor :amount, :amount_cents, :local_amount, :local_amount_cents,
```

Comment:
👍 for calling `attr_accessor` only once, but I think that I prefer to keep one attribute / line, as it seems to be easier to read. WDYT?

Btw, don't you think we should alpha sort attributes?